serializations.md

```diff
@@ -520,6 +520,7 @@ purging them from the tree.
 #### Channel close mutual transaction
 ```
 [ <channel_id>             :: id()
+, <from_id>                :: id()
```

Comment:
I see this field added.

Reply:
https://github.com/aeternity/epoch/pull/1614/files#diff-05f838352a846bddf27c4d25f75b5477R43

Ok.